script/db_backup/config.yml

```diff
@@ -0,0 +1,22 @@
+#db_backup Script Config
+
+#db
+db_user: user
```

Comment:
In YAML è più elegante usare il nesting "nativo":

```yaml
db:
    user: user
    password: pass
```

Idem per le altre chiavi.